csodiaq/csodiaq.py

```diff
@@ -27,16 +28,18 @@ def main():
         return
 
     if args['command'] == 'id':
-        lib = cif.library_file_to_dict(args['library'])
-        maxQuerySpectraToPool = queryPooling = args['query']
-        if not maxQuerySpectraToPool:
-            maxQuerySpectraToPool = np.inf
+        identifier = Identifier(args)
+        identifier.identify_library_spectra_in_queries()
+        #lib = cif.library_file_to_dict(args['library'])
```

Comment:
I'm keeping this commented code around for comparison with the old code as I continue to rework it.